### DISCUSSION TITLE
https://greendolphindance.com/2024/01/26/%E5%88%B6%E4%BD%9Chexo%E6%A0%87%E7%AD%BE%E5%A2%99%EF%BC%882%EF%BC%89/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#15] [General]
# https://greendolphindance.com/2024/01/26/%E5%88%B6%E4%BD%9Chexo%E6%A0%87%E7%AD%BE%E5%A2%99%EF%BC%882%EF%BC%89/

Back to the rough ground!

https://greendolphindance.com/2024/01/26/%E5%88%B6%E4%BD%9Chexo%E6%A0%87%E7%AD%BE%E5%A2%99%EF%BC%882%EF%BC%89/

**Comment by hanlin-ren** [comment]
草，好简单粗暴的搞法（

**Reply by greendolphindance** [reply]
但是有效（

**Comment by ouuan** [comment]
太大了，感觉 log 一下比较好，我一直是 log 都调小了两次系数（

**Reply by greendolphindance** [reply]
我去log一下！的确我某些标签异常地多（

**Reply by ouuan** [reply]
没我多（（

**Reply by greendolphindance** [reply]
笑死（秒回！

**Comment by wang7peng** [comment]
chic本身只为了一次性写好了的博客，对那种需要不定期增加的内容没有准备，所以把日期也打入url了。导致分享后下次改了日期其实原先的链接就失效了。
大姐大能不能开发一个 timeline / milestone 的功能，**就是一个向右（或者向上）的长箭头，直的弯的波的都可以，然后在上面分段，显示一些简略的内容。** 类似于分类里面那个点击子标题后显示的目录页, 找到稍微符合的是 [这样](https://kaiiiz.github.io/hugo-theme-monochrome/layouts/balloon/demo/) 的，把气球换了就差不多了，就是线条不能改。想在 chic 里也有这样的显示好像有点难度 :(

**Reply by greendolphindance** [reply]
啊啊你的链接打不开，所以看不到效果！然后我其实不会写代码，代码都是gpt写的，要不你试试能不能调教gpt呢

**Reply by wang7peng** [reply]
最近刚学 vue2，也不会 js 呢。链接出处就是 [github.com/kaiiiz](https://github.com/kaiiiz/hugo-theme-monochrome) ，是 hugo 的一个主题 。感觉它的用处还是很大的，比如有了它，大姐大你写购物分享系列就没必要标 1-2-3 后缀啦哈哈哈。然后，新年快乐！🐉🐉🐉

**Reply by greendolphindance** [reply]
你说的有道理诶！也祝你新年快乐！

**Reply by greendolphindance** [reply]
回家折腾了一下，弄出来了一个，不过可能和你设想的不太一样？anyways，请移步我今天发的文章！
